step.yml

```diff
@@ -1,44 +1,53 @@
+  - Make sure certificates are uploaded to Bitrise's **Code Signing** tab. The right provisioning profiles are automatically downloaded as part of the automatic code signing process.
 
   To configure the Step:
-
-  1. Make sure the **Project path** input points to the correct location.
-
-     By default, you do not have to change this.
-  2. Set the correct value to the **Distribution method** input. If you use the **iOS Auto Provision** Step, the value of this input should be the same as the **Distribution type** input of that Step.
-  3. Make sure the target scheme is a valid, existing Xcode scheme.
-  4. Optionally, you can define a configuration type to be used (such as Debug or Release) in the **Build configuration** input.
-
-     By default, the selected Xcode scheme determines which configuration will be used. This option overwrites the configuration set in the scheme.
-  5. If you wish to use a different Developer portal team than the one set in your Xcode project, enter the ID in the **Developer Portal team** input.
-
-  ### Troubleshooting
-
-  If the Step fails, check your code signing files first. Make sure they are the right type for your export method. For example, an `app-store` distribution method requires an App Store type provisioning profile and a Distribution certificate.
-
-  Check **Debugging** for additional options to run the Step. The **Additional options for xcodebuild command** input allows you add any flags that the `xcodebuild` command supports.
-
-  Make sure the **Scheme** and **Build configuration** inputs contain values that actually exist in your Xcode project.
-
-  ### Useful links
-
-  - https://devcenter.bitrise.io/code-signing/ios-code-signing/create-signed-ipa-for-xcode/
-  - https://devcenter.bitrise.io/code-signing/ios-code-signing/resigning-an-ipa/
-  - https://devcenter.bitrise.io/deploy/ios-deploy/ios-deploy-index/
-
-  ### Related Steps
-
-  - [Certificate and profile installer](https://www.bitrise.io/integrations/steps/certificate-and-profile-installer)
-  - [iOS Auto Provision](https://www.bitrise.io/integrations/steps/ios-auto-provision)
-  - [Deploy to iTunesConnect](https://www.bitrise.io/integrations/steps/deploy-to-itunesconnect-deliver)
+  1. **Project path**: Add the path where the Xcode Project or Workspace is located.
+  2. **Scheme**: Add the scheme name you wish to archive your project later.
+  3. **Distribution method**: Select the method Xcode should sign your project: development, app-store, ad-hoc, or enterprise.
+
+  Under **xcodebuild configuration**:
+  1. **Build configuration**: Specify Xcode Build Configuration. The Step uses the provided Build Configuration's Build Settings to understand your project's code signing configuration. If not provided, the Archive action's default Build Configuration will be used.
```

Comment:
@godrei do we want to mention these inputs here? I think these 3 are advanced features and most users should be good with the default values.